test/uk/gov/hmrc/directdebitupdateemailbackend/testsupport/ItSpec.scala

```diff
@@ -40,16 +41,25 @@ import scala.annotation.nowarn
 import scala.concurrent.duration._
 import scala.util.Random
 
+trait AppProvider {
```

Comment:
Is this needed?

Reply:
whoops - forgot about that. Was experimenting with something. Just removed it